ansible/roles/postgresql/templates/postgresql.conf.j2

```diff
@@ -201,7 +200,7 @@ vacuum_cost_delay = 20ms        # 0-100 milliseconds
 
 # These settings are ignored on a standby server
 
-#max_wal_senders = 0        # max number of walsender processes
+max_wal_senders = 3        # max number of walsender processes
```

Comment:
Got an error when testing this locally (without a hot_standby server):

FATAL:  WAL streaming (max_wal_senders > 0) requires wal_level "archive", "hot_standby", or "logical"